PennMobile/src/main/java/com/pennapps/labs/pennmobile/adapters/LaundryMachineAdapter.java

```diff
@@ -0,0 +1,192 @@
+import android.graphics.BitmapFactory;
+import android.graphics.Point;
+import android.os.SystemClock;
+import android.support.design.widget.Snackbar;
+import android.view.Display;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.CompoundButton;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.Switch;
+import android.widget.TextView;
+
+import com.pennapps.labs.pennmobile.LaundryBroadcastReceiver;
+import com.pennapps.labs.pennmobile.LaundryFragment;
+import com.pennapps.labs.pennmobile.MainActivity;
+import com.pennapps.labs.pennmobile.R;
+import com.pennapps.labs.pennmobile.classes.Laundry;
+import com.pennapps.labs.pennmobile.classes.LaundryMachine;
+
+import java.util.List;
+
+import butterknife.ButterKnife;
+
+/**
+ * Created by Jason on 11/18/2015.
+ */
+public class LaundryMachineAdapter extends ArrayAdapter<LaundryMachine> {
+
+    private List<LaundryMachine> machines;
+    private final LayoutInflater inflater;
+    private MainActivity activity;
+    private boolean wash;
+    private Laundry laundry;
+
+    public LaundryMachineAdapter(MainActivity activity, List<LaundryMachine> machines, boolean wash, Laundry laundry) {
+        super(activity, R.layout.laundry_list_item, machines);
+        this.machines = machines;
+        inflater = LayoutInflater.from(activity);
+        this.activity = activity;
+        this.wash = wash;
+        this.laundry = laundry;
+    }
+    @Override
+    public View getView(int position, View view, ViewGroup parent) {
+        if (view == null) {
+            view = inflater.inflate(R.layout.laundry_machine_list_item, parent, false);
+        }
+        ButterKnife.bind(this, view);
+        LinearLayout summary = (LinearLayout) view.findViewById(R.id.laundry_machine_list_summary);
+        RelativeLayout item = (RelativeLayout) view.findViewById(R.id.laundry_machine_list_normal);
+        StringBuilder stringBuilder = new StringBuilder();
+        if(position == 0){
+            item.setVisibility(View.GONE);
+            summary.setVisibility(View.VISIBLE);
+            ImageView imageView = (ImageView) view.findViewById(R.id.laundry_machine_iv);
+            RelativeLayout summary_rl = (RelativeLayout) view.findViewById(R.id.laundry_machine_summary_rl);
+            TextView description = (TextView) view.findViewById(R.id.laundry_building_description);
+            description.setText(laundry.name);
+            TextView availcount = (TextView) view.findViewById(R.id.laundry_machine_summary);
+            int avail = 0;
+            for(LaundryMachine machine: machines){
+                if(machine.available){
+                    avail++;
+                }
+            }
+            String str = wash? activity.getString(R.string.laundry_washer): activity.getString(R.string.laundry_dryer);
```

Comment:
style: please use consistent spacing around operators, especially in something confusing like a ternary